R/get_index_regions.R

```diff
@@ -27,15 +26,32 @@ get_index_regions <- function() {
 
   em_url <- "https://www.msci.com/our-solutions/indexes/emerging-markets"
 
-  html <- fetch_url_html(em_url)
+  em_html <- fetch_url_html(em_url)
 
-  em_countries <-
-    html %>%
-    rvest::read_html() %>%
-    rvest::html_elements(css = "#ms136-emi-010523 .index-type-container.index-type-equity") %>%
-    rvest::html_text2() %>%
-    strsplit("\n") %>%
-    unlist()
+  js_scripts <- rvest::html_elements(em_html, "div.portlet-body script")
+
+  js_script <- js_scripts[grepl("chartDataEMJson", js_scripts)]
+
+  ct <- V8::v8()
+  #parse the html content from the js output and print it as text
+  ct$eval(gsub('document.write', '', rvest::html_text2(js_script)))
```

Comment:
NB: Also consider doing the script modification into a separate variable, for clarity.
```suggestion
    render_script <- str_replace(
      pattern = 'document.write',
      replacement = '',
      string = rvest::html_text2(js_script)
    )
    ct$eval(render_script)
```